ios/browser/ui/webui/brave_account/brave_account_ui.mm

```diff
@@ -52,14 +54,28 @@
 BraveAccountUI::BraveAccountUI(web::WebUIIOS* web_ui, const GURL& url)
     : BraveAccountUIBase(ProfileIOS::FromWebUIIOS(web_ui)),
       web::WebUIIOSController(web_ui, url.host()) {
+  using Authentication = void (BraveAccountUIBase::*)(
+      mojo::PendingReceiver<brave_account::mojom::Authentication>);
   web_ui->GetWebState()->GetInterfaceBinderForMainFrame()->AddInterface(
-      base::BindRepeating(&BraveAccountUIBase::BindInterface,
+      base::BindRepeating(
+          static_cast<Authentication>(&BraveAccountUIBase::BindInterface),
+          base::Unretained(this)));
+
+  using PasswordStrengthMeter = void (BraveAccountUIBase::*)(
+      mojo::PendingReceiver<
+          password_strength_meter::mojom::PasswordStrengthMeter>);
+  web_ui->GetWebState()->GetInterfaceBinderForMainFrame()->AddInterface(
+      base::BindRepeating(static_cast<PasswordStrengthMeter>(
+                              &BraveAccountUIBase::BindInterface),
                           base::Unretained(this)));
 
   web_ui->AddMessageHandler(std::make_unique<BraveAccountUIMessageHandler>());
 }
 
 BraveAccountUI::~BraveAccountUI() {
+  web_ui()->GetWebState()->GetInterfaceBinderForMainFrame()->RemoveInterface(
```

Comment:
This doesn't seem like it should be necessary and seems fragile and error prone anyway. @kylehickinson do you know why this is here? Not an issue for this PR, but I want to look at this as a follow-up.

Reply:
https://github.com/brave/brave-browser/issues/46677